interpreter/cling/tools/plugins/clad/CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ if(MSVC)
   ExternalProject_Add(
```

Comment:
The 2 branch of the If look extremely similar.  Wouldn't be clearer to have a single call to ExternalProject_Add tweak via variables?

Reply:
@bellenot tried merging both into a single branch but did not work for some reason.